src/pages/settings/Security/TwoFactorAuth/Steps/CodesStep.js

```diff
@@ -0,0 +1,146 @@
+    const [isNextButtonDisabled, setIsNextButtonDisabled] = useState(true);
+
+    useEffect(() => {
+        if (account.recoveryCodes) {
+            return;
+        }
+        Session.toggleTwoFactorAuth(true);
+    }, []);
+
+
+    return (
+        <StepWrapper
+            title={translate('twoFactorAuth.headerTitle')}
+            stepCounter={{
+                step: 1,
+                text: translate('twoFactorAuth.stepCodes'),
+                total: 3,
+            }}
+        >
+            <ScrollView>
+                <Section
+                    title={translate('twoFactorAuth.keepCodesSafe')}
+                    icon={Illustrations.ShieldYellow}
+                    containerStyles={[styles.twoFactorAuthSection]}
+                    iconContainerStyles={[styles.ml6]}
+                >
+                    <View style={styles.mv3}>
+                        <Text>{translate('twoFactorAuth.codesLoseAccess')}</Text>
+                    </View>
+                    <View style={styles.twoFactorAuthCodesBox(dimensionsProps)}>
+                        {account.isLoading ? (
+                            <View style={styles.twoFactorLoadingContainer}>
+                                <ActivityIndicator color={themeColors.spinner}/>
+                            </View>
+                        ) : (
+                            <>
+                                <View style={styles.twoFactorAuthCodesContainer}>
+                                    {Boolean(account.recoveryCodes) &&
+                                        _.map(account.recoveryCodes.split(', '), (code) => (
+                                            <Text
+                                                style={styles.twoFactorAuthCode}
+                                                key={code}
+                                            >
+                                                {code}
+                                            </Text>
+                                        ))}
+                                </View>
+                                <View style={styles.twoFactorAuthCodesButtonsContainer}>
+                                    <PressableWithDelayToggle
+                                        text={translate('twoFactorAuth.copy')}
+                                        textChecked={translate('common.copied')}
+                                        icon={Expensicons.Copy}
+                                        inline={false}
+                                        onPress={() => {
+                                            Clipboard.setString(account.recoveryCodes);
+                                            setIsNextButtonDisabled(false);
+                                        }}
+                                        styles={[styles.button, styles.buttonMedium, styles.twoFactorAuthCodesButton]}
+                                        textStyles={[styles.buttonMediumText]}
```

Comment:
Array is not needed here, same for others with only one style passed
```suggestion
                                        textStyles={styles.buttonMediumText}
```

Reply:
Failed prop type error in the console, it expects an array